### PR TITLE
updating version for release 4.4.0

### PR DESCRIPTION
**Pull request by bambriz**
# Description

Updating to GA stable release 4.4.0

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py [hunk 1]
```diff
@@ -815,11 +815,22 @@ async def delete_all_items_by_partition_key(
...
+        operation is constrained to consume at most 10% of the total
+        available RU/s on the container each second. This helps in limiting the resources used by
+        this background task.
 
         :param partition_key: Partition key for the items to be deleted.
         :type partition_key: Any
+        :keyword str pre_trigger_include: trigger id to be used as pre operation trigger.
+        :keyword str post_trigger_include: trigger id to be used as post operation trigger.
+        :keyword str session_token: Token for use with Session consistency.
+        :keyword dict[str,str] initial_headers: Initial headers to be sent as part of the request.
```
Comment (simorenoh) [inline sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py:829]: let's remove initial_headers from the documentation for this method and for patch api actually, sorry I missed the fact I added it there as well - we should move towards not supporting this option for users and having them instead pass us the options to set the headers within the SDK

**Review by simorenoh — COMMENTED** [review]
Awesome, thanks for this Bryan! Left one small comment on removing the kwarg documentation for initial_headers for this method and the patch method - please make those changes, thanks!

File: sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py [hunk 2]
```diff
@@ -815,11 +815,22 @@ async def delete_all_items_by_partition_key(
         partition_key: Union[str, int, float, bool],
         **kwargs: Any
     ) -> None:
-        """Exposes an API to delete all items with a single partition key without the user having
-                 to explicitly call delete on each record in the partition key.
+        """The delete by partition key feature is an asynchronous, background operation that allows you to delete all
+        documents with the same logical partition key value, using the Cosmos SDK. The delete by partition key
+        operation is constrained to consume at most 10% of the total
+        available RU/s on the container each second. This helps in limiting the resources used by
+        this background task.
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py:822]: May be we can also link to our public docs for this and patch API as well if not already.

**Review by simorenoh — APPROVED** [review]

File: sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py (hunk 2 above)
Comment (simorenoh) [reply sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py:821]: The links for both are in the Changelog with their respective entries.

**Pull request merged**